### PR TITLE
Huge fixes in 2x2 and 3x3 scramble generators

### PR DESCRIPTION
**Pull request by LucasAlfare**
Once the proporse of random sequences in a timer is just work as scrambles, doesn't makes too sense treat then as a simple "shuffle" algorithm.

Here, now, are implemented a simple algorithm to generate those sequences in order to make then useful in a practical context.

Feel free to ask any questions about the implementation or even architecture.

### CODE REVIEW

File: lib/Shufflers.py [hunk 1]
```diff
@@ -9,51 +8,73 @@
...
-                if m == 5:
-                    old = m
-                    moves.append('L') # Left
-                    break
-                if m == 6:
-                    old = m
-                    moves.append('B') # Back
-                    break
+        moveA = ""
+        moveB = ""
```
Comment (Samuel-de-Oliveira) [inline lib/Shufflers.py]: I noticed that  the moveB var in line 23 don't need to be declare, so if you remove this line the program will work normally.

**Review by Samuel-de-Oliveira — CHANGES_REQUESTED** [review]
Great, but I recommend you change a thing after merge.

File: lib/Shufflers.py (hunk 1 above)
Comment (LucasAlfare) [reply lib/Shufflers.py]: No problem, changes pushed. Btw, this is just a clean up revision.

**Comment by Samuel-de-Oliveira** [comment]
Thank you!

**Pull request merged**